src/sentry/types/issues.py

```diff
@@ -14,13 +14,18 @@ class GroupType(Enum):
     PERFORMANCE_CONSECUTIVE_DB_OP = 1007
     PERFORMANCE_FILE_IO_MAIN_THREAD = 1008
     PERFORMANCE_N_PLUS_ONE_API_CALLS = 1010
+    PROFILE_BLOCKED_THREAD = 2000
 
 
 class GroupCategory(Enum):
     ERROR = 1
     PERFORMANCE = 2
+    PROFILE = 3
 
 
+GROUP_CATEGORIES_CUSTOM = (GroupCategory.ERROR, GroupCategory.PERFORMANCE)
```

Comment:
`*_CUSTOM_EMAIL` maybe?

Reply:
I guess I was thinking that if you customize email you _may_ customize more integrations' content.

Reply:
Right, but I think that may need to be separate (or we include the things that were customized in this structure?)